extensions/kafka/src/main/java/io/deephaven/kafka/ingest/KafkaStreamPublisher.java

```diff
@@ -74,63 +79,57 @@ private KafkaStreamPublisher(
     }
 
     public static ConsumerRecordToStreamPublisherAdapter make(
```

Comment:
No need, regarding Python. This is currently only ever used from Java, internal to KafkaTools. Really, the parameters struct was just to avoid repeating a bunch of ugly code, and I had IntelliJ generate it automatically. I tried the builder on for size initially, but it was even more gratuitous code. Let me take one more crack at seeing if it makes things better.

Reply:
OK, I added a bunch of code and a builder. It's cleaner, but I didn't clean up every bit of badness, either.